sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/WritableProperty.cs

```diff
@@ -10,7 +10,12 @@ namespace Azure.DigitalTwins.Core.Serialization
     /// The ModelProperties dictionary on <see cref="DigitalTwinMetadata"/>.
     /// </summary>
     /// <remarks>
+    /// <para>
     /// A writable property is one that the service may request a change for from the device.
+    /// </para>
```

Comment:
para means paragraph?

Reply:
https://review.docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments?branch=master